test/integrations/deployers/DeployerCoordinator.t.sol

```diff
@@ -320,6 +320,85 @@ contract DeployerCoordinatorTest is HyperdriveTest {
         assertEq(deployment.target4, targets[3]);
     }
 
+    function test_deployTarget_frontrun() external {
+        // Deploy a target0 instance as deployer (should succeed).
+        coordinator.deployTarget(DEPLOYMENT_ID, config, new bytes(0), 0, SALT);
+
+        vm.stopPrank();
+        vm.startPrank(bob);
+        // Try to frontrun the target1 deployment with bob
+        vm.expectRevert(
+            IHyperdriveDeployerCoordinator.DeploymentDoesNotExist.selector
+        );
+        coordinator.deployTarget(DEPLOYMENT_ID, config, new bytes(0), 1, SALT);
+        vm.stopPrank();
```

Comment:
This would have passed before. What I think would make a better test is to have `bob` try to deploy `target0` with `DEPLOYMENT_ID` and `SALT`. Next, we should have `alice` deploy `target0` with the same `DEPLOYMENT_ID` and `SALT` to illustrate that Alice can't be front-run by Bob.

Reply:
Since the below test can be used to test that front-running will fail, we can get rid of this test case.